game-core/src/test/java/games/strategy/triplea/delegate/battle/casualty/AaCasualtySelectorTest.java

```diff
@@ -417,4 +419,606 @@ void hitsLessThanPlanesKillsAccordingToTheRolledDice() {
           is(List.of(planes.get(0), planes.get(2), planes.get(4))));
     }
   }
+
+  @Nested
+  class LowLuckCasualties {
```

Comment:
I want to say it's pretty amazing there was no coverage on this before, then again with 27% test coverage, maybe not so much : )

Thanks for adding coverage!